internal/services/appservice/helpers/windows_web_app_schema.go

```diff
@@ -828,7 +813,12 @@ func (s *SiteConfigWindows) ExpandForUpdate(metadata sdk.ResourceMetaData, exist
 		}
 		expanded.Cors = cors
 	}
+
 	if metadata.ResourceData.HasChange("site_config.0.auto_heal_setting") {
+		expanded.AutoHealEnabled = pointer.To(false)
```

Comment:
again here, we should double check the logic, but I believe we'll need to not set `false` at the top of the function or we'll possibly switch off auto-heal unexpectedly?

Reply:
Thanks Steve, my first thought is to check the situation like:
1. the `auto_heal` is enabled
2. there is a change to another property under `site_config`

We can't turn the auto heal off under the above mentioned situation during the app update, but change the auto heal state only when the change is targeted to `auto_heal` setting.

Does it makes sense to you?